android/app/src/main/java/app/organicmaps/widget/menu/MyPositionButton.java

```diff
@@ -77,8 +80,20 @@ public void update(int mode)
     ImageViewCompat.setImageTintList(mButton, ColorStateList.valueOf(ThemeUtils.getColor(context, colorAttr)));
     updatePadding(mode);
 
-    if (image instanceof AnimationDrawable)
-      ((AnimationDrawable) image).start();
+    if (mode == LocationState.PENDING_POSITION)
+    {
+      final RotateAnimation rotate = new RotateAnimation(0, 360,
+              Animation.RELATIVE_TO_SELF, 0.5f, Animation.RELATIVE_TO_SELF,
+              0.5f);
```

Comment:
The formatting was strange... A bit longer lines (120-140 chars) are perfectly ok, they're easier to read on modern screens.
```suggestion
      final RotateAnimation rotate = new RotateAnimation(0, 360, Animation.RELATIVE_TO_SELF, 0.5f, Animation.RELATIVE_TO_SELF, 0.5f);
```

Reply:
fixed, turns out `Animation.RELATIVE_TO_SELF` is just = `1`, so it all fits on one line anyway 😅

Reply:
Using descriptive text constants is more readable than just numbers )

Reply:
fair enough, fixed :)